pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/rbac/crd_editor_role.go

```diff
@@ -65,7 +65,14 @@ func (f *CRDEditorRole) SetTemplateDefaults() error {
 	return nil
 }
 
-const crdRoleEditorTemplate = `# permissions for end users to edit {{ .Resource.Plural }}.
+const crdRoleEditorTemplate = `# This rule is not used by the project {{ .ProjectName }} itself.
+# It is provided to allow the cluster admin to help manage permissions for users.
+#
+# Allows users to modify all custom resources for {{ .ProjectName }} on the cluster.
```

Comment:
```suggestion
# Allows users to modify {{ .Resource.QualifiedGroup }} on the cluster.
```

I think we could be more clear.
We scaffold a file for each API/CRD
@damsien could we do this update in the 3 scenarios?